src/diceBot/BadLife.rb

```diff
@@ -67,7 +67,7 @@ def rollDiceCommand(command)
   end
 
   def judgeDice(command)
-    unless (m = /(\d+)?(BAD|BL|GL)([\+\-\d+]*)((C|F)([\+\-\d+]*)?)?((C|F)([\+\-\d+]*))?(\@([\+\-\d+]*))?(\!(\D*))?/i.match(command))
+    unless (m = /(\d+)?(BAD|BL|GL)([\+\-\d]*)((C|F)([\+\-\d]*)?)?((C|F)([\+\-\d]*))?(\@([\+\-\d]*))?(\!(\D*))?/i.match(command))
```

Comment:
文字クラス内で `-` から始めるようにすれば、エスケープ不要な部分がありました。

```suggestion
    unless (m = /(\d+)?(BAD|BL|GL)([-+\d]*)((C|F)([-+\d]*)?)?((C|F)([-+\d]*))?(\@([-+\d]*))?(\!(\D*))?/i.match(command))
```